### PR TITLE
Migrate splash

### PR DESCRIPTION
**Pull request by mikeyrayvon**

### CODE REVIEW

**Comment by varffs** [comment]
@mixcas plz just check the main.js as everything else is fine

File: js/main.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
 /* jshint browser: true, devel: true, indent: 2, curly: true, eqeqeq: true, futurehostile: true, latedef: true, undef: true, unused: true */
-/* global $, jQuery, document, Modernizr */
+/* global $, jQuery, document, Modernizr, Site */
 Site = {
   mobileThreshold: 1008,
   init: function() {
     var _this = this;
 
+    _this.windowHeight = $(window).height();
```
Comment (mixcas) [inline js/main.js:8]: This needed?
Cuz they get new values in `onResize`

Comment (varffs) [inline js/main.js:8]: Needed for layout before a resize

Comment (mikeyrayvon) [inline js/main.js:8]: we should call resize in init

Comment (varffs) [inline js/main.js:8]: _does not compute_

Comment (mikeyrayvon) [inline js/main.js:8]: oh i see whats happeneing nvm

**Comment by mixcas** [comment]
@varffs i really disagree with caching window size. It's a value we are getting directly from the browser. Aside from the fact that `Site.onResize()` is doing other functionality besides caching those values, this feels like the browser's job. Also, `window` is a global too!

Anyway, if performance is the motif for this caching I ran some tests 🎱 

```
console.time('Directly from jQuery');
var x = jQuery(window).width() * 2;
console.timeEnd('Directly from jQuery');

console.time('Directly from Browser');
var y = window.innerWidth * 2;
console.timeEnd('Directly from Browser');

a = jQuery(window).width();
console.time('Cached');
var z = a *2;
console.timeEnd('Cached');
```

Directly from jQuery: 0.116ms
Directly from Browser: 0.014ms
Cached: 0.006ms

Run 10x

```
console.time('Directly from jQuery');
for(var i = 0; i <= 10; i++) {
  var x = jQuery(window).width() * 2;
}
console.timeEnd('Directly from jQuery');

console.time('Directly from Browser');
for(var i = 0; i <= 10; i++) {
  var y = window.innerWidth * 2;
}
console.timeEnd('Directly from Browser');

a = jQuery(window).width();
console.time('Cached');
for(var i = 0; i <= 10; i++) {
  var z = a *2;
}
console.timeEnd('Cached');
---
Directly from jQuery: 0.352ms
Directly from Browser: 0.030ms
Cached: 0.019ms
```

Nice, cached is definitely faster, 36% faster

Lets do it 1,000,000x

```
console.time('Directly from jQuery');
for(var i = 0; i <= 1000000; i++) {
  var x = jQuery(window).width() * 2;
}
console.timeEnd('Directly from jQuery');

console.time('Directly from Browser');
for(var i = 0; i <= 1000000; i++) {
  var y = window.innerWidth * 2;
}
console.timeEnd('Directly from Browser');

a = jQuery(window).width();
console.time('Cached');
for(var i = 0; i <= 1000000; i++) {
  var z = a *2;
}
console.timeEnd('Cached');
---
Directly from jQuery: 19828.276ms
Directly from Browser: 1332.096ms
Cached: 1055.545ms
```

Ok 20% faster.

Well, we definitely know that calling `$(window).width()` by itself is way slower than calling `window.innerWidth`. So maybe we should take that on account for future code. Calling directly from `window` is also easier to type cuz requires less symbols.

Going back to the real issue, I feel the performance difference is not good enough to justify tangling the code. Who knows, maybe calling a value inside a global var (ex. `Site.widowWidth`) has other issues that these tests don't account for.

One thing is for sure, calling `window.innerWidth` is direct, almost as fast as caching the value and doesn't require extra code to be updated, ergo cleaner code == better world 🌏 .

**Comment by varffs** [comment]
You know there are tools for this kind of testing: https://benchmarkjs.com/

Clearly browser is way faster than jQuery [but the jQuery code isn't just a wrapper clearly so I wonder what is going on there?] but 20% is actually still quite a good performance increase on top of that? Considering that `Site.windowHeight` is used on a scroll event which fires a lot.

I'm ambivalent really but I do kinda feel that your testing evidence shows that caching the value is worthwhile rather than not. 

And I feel the cleanliness argument doesn't hold up too well either considering that if this were to be a consistent pattern with `Site` always having a resize and setting these cache values the difference in the rest of the code is between `window.innerWidth` & `Site.windowWidth` which is more or less the same.

**Comment by mixcas** [comment]
Anywhere where we are/ will be using `Site.windowWidth` we could be finely using `window.innerWidth` without having to be worrying about caching this value. It's the window size we are talking about ffs. I would agree to cache other values, like network requests, element sizes or positions but really not this. 

The difference sounds good as 20% but its really 0.008ms (milliseconds!) and really my test is not accounting for having a global var updated and retrieving that value from inside another object (doing what the browser already does)

Using `Site.windowWidth` is kinda ugly too, and it doesn't really get any advantages from being a property of `Site` since it will be called from other objects as a global, it not like we are inheriting anything.

I rather have less and cleaner code than worry about some 0.008ms performance improvement. If we were really to make our code faster we should be looking at ditching jquery for good and worrying about our iterations, etc… not really about caching the window size.

**Comment by varffs** [comment]
As I said ambivalent but I don't really see your arg tbh. Can't see how that value is more or less ugly than directly addressing the window. And can't see how we have to `worry` about caching if it is a pattern and we can always depend on that value being set. But if you want to strip it out and get rid of the cache

**Comment by mixcas** [comment]
When i say "worry" i mean the code running to keep those values updated.

`window` is world wide web known and pretty clear. It's something magic inside the browser and "we can always depend on that value being set" without even stopping to think for a sec "what is this? oh yeah the window size".

I'm gonna merge this cuz i think we are wasting time dicussing this… **but i know the day will come, the day when i will says _i told you so_ MUAAAHAHAHAHAHAHAHAH** 😈

**Comment by varffs** [comment]
Go on then merge it!

**Comment by mixcas** [comment]
@varffs take a chill pill brah

**Comment by mikeyrayvon** [comment]
man its getting hot in here

**Comment by mixcas** [comment]
Guys when deleting/moving/renaming files plz use proper git commands (idk how to that on GUI) to avoid weird conflicts like this

```
CONFLICT (modify/delete): index.html deleted in HEAD and modified in master. Version master of index.html left in tree.
CONFLICT (rename/delete): img/casabarragan_red.jpg deleted in HEAD and renamed in master. Version master of img/casabarragan_red.jpg left in tree.
Auto-merging css/site.styl
CONFLICT (content): Merge conflict in css/site.styl
CONFLICT (modify/delete): css/site.min.css deleted in HEAD and modified in master. Version master of css/site.min.css left in tree.
Auto-merging css/site.css
CONFLICT (content): Merge conflict in css/site.css
Automatic merge failed; fix conflicts and then commit the result.
```

**Pull request merged**